Server/Quality_of_Life_changer.WebApi/Controllers/UserProfileController.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Quality_of_Life_changer.WebApi.Controllers;
+
+using Contracts.Commands;
+using Contracts.Exceptions;
+using FluentValidation;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Model.UserProfileModel;
+
+[Route("api/[controller]")]
```

Comment:
either remove this Route, or move `user/{userId}/profile` part here.
otherwise, you're getting `api/userprofile/user/{id}/profile/calendars`

Reply:
if a delete `[controller]` in this route, and i will to delete in other controllers `[controller]`?

Reply:
it's better to move `user/{userId}/profile` part here, since it's a user profile controller.
but if it's impossible, then you can remove `[controller]` part